### PR TITLE
Updated Alipay to use config file and readme

### PR DESCRIPTION
**Pull request by JReissmueller**

### CODE REVIEW

File: language/en_us/alipay.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 // Main
 $lang['Alipay.name'] = 'Alipay';
+$lang['Alipay.description'] = '';
```
Comment (tysonphillips) [inline language/en_us/alipay.php]: You should probably set some type of description relevant to this gateway, like the region it is mostly used in, what actions it supports, etc.

File: config.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	"version": "1.1.0",
+	"name": "Alipay.name",
+	"description": "Alipay.description",
+	"authors": [
+		{
+			"name": "Phillips Data, Inc.",
+			"url": "http://www.blesta.com"
+		}
+	],
+	"currencies": ["AUD"],
```
Comment (tysonphillips) [inline config.json]: How come the number of currencies changed from 10 to 1?

File: alipay.php [hunk 3]
```diff
@@ -33,53 +23,15 @@ class Alipay extends NonmerchantGateway
      */
     public function __construct()
     {
+        $this->loadConfig(dirname(__FILE__) . DS . 'config.json');
```
Comment (tysonphillips) [inline alipay.php:26]: The link to the documentation on line 6 doesn't appear to work anymore, so might as well update it to wherever their current documentation is

**Review by tysonphillips — CHANGES_REQUESTED** [review]

File: language/en_us/alipay.php (hunk 1 above)
Comment (JReissmueller) [reply language/en_us/alipay.php]: Updated

**Review by tysonphillips — APPROVED** [review]

**Pull request merged**